normalize_repos/normalize_repos.py

```diff
@@ -7,14 +7,17 @@
 
 # Standard library
 import logging
+import sys,os
 
 # noinspection PyUnresolvedReferences
 import set_up_logging
 
 # Local/library specific
 import log
-from utils import get_cc_organization, set_up_github_client
-from get_labels import get_labels
+sys.path.append((os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
I'm not a big fan of modifying system paths, is there no other option?

Reply:
I found two methods one was this .The other was to create an <code>__init__py</code> file in each directory so that we can use relative paths .